### PR TITLE
3 ab starting on UI

### PR DESCRIPTION
**Pull request by aborov**

### CODE REVIEW

**Review by aizatibraimova — APPROVED** [review]
Everything looks ok

**Review by jniewiel — APPROVED** [review]
Looks good!

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -6,11 +6,25 @@
...
 
+    <%= render partial: "shared/cdn_assets" %>
+
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
   </head>
 
   <body>
-    <%= yield %>
+    <%= render partial: "shared/navbar" %>
```
Comment (JoseMDO) [inline app/views/layouts/application.html.erb:16]: Same as above

File: app/views/shared/_navbar.html.erb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav mr-auto">
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+            Resources
+          </a>
+          <div class="dropdown-menu">
+            <%= link_to "Photos", photos_path, class: "dropdown-item" %>
```
Comment (JoseMDO) [inline app/views/shared/_navbar.html.erb:14]: Great job using the new helper methods Aleksei!

File: lib/tasks/dev.rake [hunk 3]
```diff
@@ -10,12 +10,16 @@ task sample_data: :environment do
...
+
+
+  usernames.each do |username|
     User.create(
-      email: "#{name}@example.com",
+      email: "#{username}@example.com",
       password: "password",
-      username: name,
+      username: username.downcase,
       private: [true, false].sample,
```
Comment (JoseMDO) [inline lib/tasks/dev.rake:22]: Great job creating sample data!

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -6,11 +6,25 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    <%= render partial: "shared/cdn_assets" %>
```
Comment (JoseMDO) [inline app/views/layouts/application.html.erb:9]: Nice job putting your CDNs in a partial!

**Review by JoseMDO — APPROVED** [review]

**Pull request merged**